packages/grid/_modules_/grid/constants/envConstants.ts

```diff
@@ -0,0 +1,8 @@
+import { localStorageAvailable } from '../utils/utils';
+
+const force =
+  localStorageAvailable() && window.localStorage.getItem('EXPERIMENTAL_ENABLED') != null;
+
+export const EXPERIMENTAL_ENABLED =
+  process.env.EXPERIMENTAL_ENABLED !== undefined &&
```

Comment:
so do you need `process.env.EXPERIMENTAL_ENABLED` defined locally?

Reply:
What do you mean by 1. "need", 2. "you", and 3. "locally"?

The idea here is to get the resolution to `false` in the developers' bundle while allowing us to force the activation of the experimental features.